src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -182,7 +182,10 @@ def _parse(self):
             self.disk_api = dev
             device_type = dev.get('TYPE', '')
             # always check is this is an lvm member
-            if device_type in ['part', 'disk']:
+            valid_types = ['part', 'disk']
+            if os.environ.get("CEPH_VOLUME_USE_LOOP_DEVICES", False):
```

Comment:
I would change this name
```suggestion
            if os.environ.get("CEPH_VOLUME_ALLOW_LOOP_DEVICES", False):
```